contracts/Aavegotchi/facets/WhitelistFacet.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.1;
+
+import {LibMeta} from "../../shared/libraries/LibMeta.sol";
+import {Modifiers, WHITELIST_LIMIT} from "../libraries/LibAppStorage.sol";
+
+contract WhitelistFacet is Modifiers {
+    event WhitelistCreated(uint256 indexed whitelistId, address indexed owner, address[] addresses);
+
+    event WhitelistUpdated(uint256 indexed whitelistId, address indexed owner, address[] addresses);
+
+    function createWhitelist(address[] memory _whitelistAddresses) external {
+        uint256 whitelistLength = _whitelistAddresses.length;
+        require(whitelistLength > 0, "WhitelistFacet: Whitelist length should be larger than zero");
+        require(whitelistLength <= WHITELIST_LIMIT, "WhitelistFacet: Whitelist length exceeds limit");
+        for (uint256 i; i < whitelistLength; i++) {
```

Comment:
This loop is gas inefficient.  In the future gas prices on Polygon may increase.   I suggest checking for `address(0)` within the loop in the `_addAddressesToWhitelist` function.

Reply:
Is there a specific reason to check if an address is 0?     If not, then I would not check for it.    0 is just one possible incorrect address that someone could use by mistake.

Reply:
Is this still an issue? @mudgen I can't seem to find the code you're referring to.

Reply:
Yes, it is still an issue.  It is line 16 above, the for loop.   I would remove the for loop.   I don't see a reason to check for 0 addresses.